### PR TITLE
fix some tyr tasks after integration tests

### PR DESCRIPTION
**Pull request by francois-vincent**
one case of squared roles, some redundant executions, symlink should be idempotent too

### CODE REVIEW

File: fabfile/utils.py [hunk 1]
```diff
@@ -210,6 +213,21 @@ def decorated(*args, **kwargs):
     return decorated
 
 
+def execute_flat(task, *args, **kwargs):
```
Comment (antoine-de) [inline fabfile/utils.py]: can you explain a bit what this is doing (and especially why) ?

Comment (francois-vincent) [inline fabfile/utils.py]: this will ignore @roles() and other deco if an env.host_string is already defined (at execution time)

Comment (francois-vincent) [inline fabfile/utils.py]: OK, I added a detailed docstring :)

Comment (antoine-de) [inline fabfile/utils.py]: ok thanks

**Pull request merged**